src/libraries/Microsoft.Bcl.Numerics/src/Microsoft.Bcl.Numerics.csproj

```
@@ -0,0 +1,37 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;$(NetFrameworkMinimum);netstandard2.1</TargetFrameworks>
+    <IsPackable>true</IsPackable>
+    <NoWarn>$(NoWarn)</NoWarn>
+
+    <!-- This assembly should never be placed inbox as it is only for downlevel compatibility. -->
+    <PackageDescription>Provides the System.MathF for .NET Standard 2.0</PackageDescription>
+    <!-- Disabling baseline validation since this is a brand new package.
+         Once this package has shipped a stable version, the following line
+         should be removed in order to re-enable validation. -->
+    <DisablePackageBaselineValidation>true</DisablePackageBaselineValidation>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <OmitResources Condition="$([MSBuild]::GetTargetFrameworkIdentifier('$(TargetFramework)')) == '.NETCoreApp'">true</OmitResources>
```

As you removed the .NETCoreApp TFM, this should now be something different:

```suggestion
    <OmitResources Condition="'$(TargetFramework)' == 'netstandard2.1'">true</OmitResources>
```

---

Now that I have removed the Strings.resx file I can just rmeove this entirely.